pkg/process/analyze.go

```diff
@@ -38,8 +41,8 @@ type TargetDetails struct {
 }
 
 type AllocationDetails struct {
-	StartAddr uint64
-	EndAddr   uint64
+	Addr    uint64
```

Comment:
minor: Not sure this needed to be renamed, I think `StartAddr` makes sense as it's the start start of the memory allocation with the other property describing the end address.